README.md

```diff
@@ -36,47 +37,59 @@ $ brew install kubernetes-helm
 $ helm init
 ```
 
-Add the minikube IP to /etc/hosts:
-```
-$ sudo sh -c 'echo "$(minikube ip) dispatch.local api.dispatch.local" >> /etc/hosts'
-```
+### Configure Authentication
+Configure an OAuth2 Client App at GitHub with instruction [How to Create OAuth Client
+App](https://vmware.github.io/dispatch/guides/create-oauth-client-app.html)
```

Comment:
when the host is optional now and we use IPs instead, what``Homepage URL`` and ``Authorization callback URL`` should be provided in github oauth2 app portal.

haven't test that with github, but you may have already.

Reply:
Using IP address instead of hostname will work. I have specified it in the *Update the Callback URL of your OAuth2 Client App* section. Probably will add a note to the Oauth2 Guide as well.